### ISSUE TITLE
Other Request for Funding: Backup Jupyterhub for Astropy workshop

### ISSUE THREAD

**Issue by stargaser** [astropy/astropy-project#367]
### Full name

David Shupe

### Purpose

In past Astropy workshops at AAS meetings, we have relied upon mybinder.org to provide a backup workshop environment for those participants who were not able to install the Python environment on their laptops before the event. With the withdrawal of support from Google earlier this year, mybinder.org has become much slower and less reliable.

This funding request is for provisioning a Jupyterhub with the Astropy workshop environment and notebooks pre-installed. We still prefer that participants install their own environment on their laptops, so this will serve as a backup. We would like to deploy this Jupyterhub for testing in mid-December, scale it up to accommodate workshop participants (max 50) at the AAS 243 workshop on January 7, 2024, leave it up for a week afterwards, and then shut it down for good.

We plan to follow the [Zero to Jupyterhub guide](https://z2jh.jupyter.org) for AWS EKS due to familiarity.

### Budget

I found it difficult to make a precise estimate using an online AWS calculator. The result it gave me was $76 for one month for one AWS EKS cluster.

Given the uncertainties, I would like to make this request for $300 USD, and to be reimbursed for whatever the actual amount turns out to be (likely to be less than this amount).

**Comment by kelle** [comment]
The finance committee approves this request.  Please [submit your expenses to the Moore project on open collective](https://opencollective.com/astropy/projects/astropy-moore-8435/expenses/new).

**Comment by stargaser** [comment]
[Submitted to opencollective](https://opencollective.com/astropy/projects/astropy-moore-8435/expenses/189574).

@hamogu was spot-on. In the end we used SciServer as the backup for the workshop participants who didn't install the workshop environment. I counted at least three people using it. Our colleagues who run the [NAVO workshop](https://github.com/NASA-NAVO/navo-workshop) had an image set up on SciServer, and we just told people to do `pip install -r requirements.txt` into there.

My attempts for make a Jupyterhub instance stalled out, because I could not get my power-user account to have the proper credentials to control the Kubernetes cluster from the command line. SciServer is a better alternative, since the Astropy Project doesn't have to maintain it, and workshop participants can keep using it long after the workshop is over.
